package.json

```diff
@@ -11,10 +11,10 @@
     "fs": false
   },
   "dependencies": {
-    "superagent": "3.8.3"
```

Comment:
Hey @ograu this introduced that problem with the outdated version of superagent again.

We have to make sure to exclude any changes to package.json before committing changes after generating the client.

I know this is super annoying, I'm still trying to update the generator so we dont' have to worry about this each time.

Reply:
@oponder you're right, sorry I forgot, I am going checkout master's package.json now

Reply:
Ok, this is merged, I'm doing a new PR...